src/stylesheets/base.css

```diff
@@ -10,6 +11,11 @@
   --bestofjsPurple: #9c0042;
   --boxBorderColor: #cecece;
   --iconColor: #fa9e59;
+  /* fonts */
+  --bodyFontFamily: 'Roboto Slab', sans-serif;
+  --headingFontFamily: 'Open Sans', 'Source Sans Pro';
```

Comment:
Might want `sans-serif` as a fallback font for the headings too?

Reply:
Good catch, it was a miss, fixed here https://github.com/bestofjs/bestofjs-webui/pull/110/commits/526dd8d0f6e5554fe430ebfe3aeec4e9fda7ce3c